data/localized/de_patch/text/quests/10_od_nua/10_qst_master_below.stringtable

```diff
@@ -98,7 +98,7 @@ Od Nua wurde von den Leuten seiner Zeit für seine Taten umgebracht, aber die Bu
     </Entry>
     <Entry>
       <ID>10006</ID>
-      <DefaultText>Um die Füße der riesigen Adra-Statue fand ich den ‚Meister in der Tiefe‘&#160;– einen mächtigen und uralten Drachen. Lange bevor Od Nua fiel, machte sich dieser Drache diese große Schlucht zum Zuhause. Im Laufe der Zeit wurde er physisch an den Adra gebunden und benutzte ihn, um mit den Seelen an der Oberfläche zu kommunizieren und um ihre Leben und ihre Geschichte zu beobachten. Nun aber ist er es Leid, nur ein passiver Beobachter zu sein, und möchte aus seinem Gefängnis ausbrechen. Er hat eine geeignete Kandidatin gefunden, die als Gefäß für seine Seele herhalten soll: Falanroed, das letzte Mitglied eines Drachentöterordens. Als ersten Schritt hat mit der Drache eine seiner Adra-Schuppen gegeben und mich gebeten, sie auf die Handfläche der Statue auf der höchsten Ebene der Endlosen Pfade zu legen, wo einst Maerwald lebte.</DefaultText>
+      <DefaultText>Um die Füße der riesigen Adra-Statue fand ich den ‚Meister in der Tiefe‘&#160;– einen mächtigen und uralten Drachen. Lange bevor Od Nua fiel, machte sich dieser Drache diese große Schlucht zum Zuhause. Im Laufe der Zeit wurde er physisch an den Adra gebunden und benutzte ihn, um mit den Seelen an der Oberfläche zu kommunizieren und um ihre Leben und ihre Geschichte zu beobachten. Nun aber ist er es Leid, nur ein passiver Beobachter zu sein, und möchte aus seinem Gefängnis ausbrechen. Er hat eine geeignete Kandidatin gefunden, die als Gefäß für seine Seele herhalten soll: Falanroed, das letzte Mitglied eines Drachentöterordens. Als ersten Schritt hat mit der Drache eine seiner Adraschuppen gegeben und mich gebeten, sie auf die Handfläche der Statue auf der höchsten Ebene der Endlosen Pfade zu legen, wo einst Maerwald lebte.</DefaultText>
```

Comment:
Adra-Statue etc wollte ich irgendwann später machen. Dann aber global. Der Strich hinter Tiefe? Ist vermutlich im englischen Text auch drinnen. Und Meister der Tiefe als „Meister in der Tiefe“ ?